packages/backend/src/house/house.util.ts

```diff
@@ -0,0 +1,19 @@
+// Generate random house code
+// Inspired by:
+// https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript/1349426#1349426
+
+import { Injectable } from '@nestjs/common';
+@Injectable()
+export class HouseUtil {
+  constructor() {}
+  generateString(length) {
+    const characters =
```

Comment:
This may be a cleaner way:

```ts
Math.random().toString(36).substr(2, length).toUpperCase()
```

https://stackoverflow.com/a/38622545/13802895

Reply:
can we also please use the crypto node library for secure randomness

Reply:
https://nodejs.org/api/crypto.html#cryptorandomintmin-max-callback